monitoring/mock_uss/msgsigning/routes_msgsigning.py

```diff
@@ -0,0 +1,31 @@
+import flask
+
+from monitoring.mock_uss import webapp, config
+from monitoring.mock_uss.msgsigning.database import db
+from loguru import logger
+import os
+
+
+@webapp.route(
+    "/mock/msgsigning/.well-known/uas-traffic-management/pub.der", methods=["GET"]
+)
+def get_public_key():
+    public_key_file_location = os.path.join(
+        webapp.config.get(config.KEY_CERT_BASE_PATH), db.value.public_key_name
+    )
+
+    logger.info("Retrieving public key file from {}".format(public_key_file_location))
+
+    return flask.send_file(public_key_file_location)
+
+
+@webapp.route("/mock/msgsigning/set-keypair", methods=["POST"])
```

Comment:
We generally want to follow the [RESTful design philosophy](https://www.redhat.com/en/topics/api/what-is-a-rest-api) and make endpoints resource-based rather than action-based.  In this case, I would expect the resource to be the keypair mock_uss is using, so I would expect the endpoint corresponding to that resource to be something like `/mock/msgsigning/keypair`.  POST is the verb usually used to create a new item in a collection, so I don't think it fits here (there is no collection and we're not creating a new item).  PUT is the verb to upsert a particular, specific resource.  There is no ID on this resource and that's unusual, but I think appropriate since mock_uss can only have one active keypair, but the thing we want to do here is to directly modify a specific resource (the single active keypair resource), so it seems like PUT would be the appropriate verb.

```suggestion
@webapp.route("/mock/msgsigning/keypair", methods=["PUT"])
```